src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -111,7 +176,7 @@ public override async Task<bool> CanChangeNamespaceAsync(Document document, Synt
             // Make sure given namespace name is valid, "" means global namespace.
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
             if (targetNamespace == null
-                || (targetNamespace.Length > 0 && !targetNamespace.Split(s_dotSeparator).All(syntaxFacts.IsValidIdentifier)))
+                || (targetNamespace.Length > 0 && !targetNamespace.Split(s_dotSeparator).All<string>(syntaxFacts.IsValidIdentifier)))
```

Comment:
```suggestion
                || (targetNamespace.Length > 0 && !targetNamespace.Split(s_dotSeparator).All(syntaxFacts.IsValidIdentifier)))
```

Reply:
I'm not actually sure when all of these generic arguments got added... ugh...